### PR TITLE
removed box-shadow on clicking the animated button #69

### PR DESCRIPTION
**Pull request by shubhampandeyhld**
animated button will no longer have the default box-shadow.

Issue: box-shadow on click button

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Hi there! I need the fix to be in the `src/components/animated/_click.less` file. Nothing in the assets files should be modified. Just add a `box-shadow: none;` to click buttons on `:active`

**Comment by shubhampandeyhld** [comment]
I have made the changes

File: src/components/animated/_click.less [hunk 1]
```diff
@@ -4,6 +4,10 @@
   &:active {
     box-shadow: 0px 2px 0px @lightGray;
   }
+  .click-btn:active{
```
Comment (shahednasser) [inline src/components/animated/_click.less:7]: remove this

File: src/components/animated/_click.less [hunk 2]
```diff
@@ -4,6 +4,10 @@
   &:active {
```
Comment (shahednasser) [inline src/components/animated/_click.less:4]: Change this to:
```
&:active,
&:focus
```

File: assets/js/index.js [hunk 3]
```diff
@@ -1,4 +1,101 @@
 $(document).ready(function(){
+$.fn.isOnScreen = function(){
```
Comment (shahednasser) [inline assets/js/index.js:2]: This entire file shouldn't be changed. You can return it to how it was with `git checkout assets/js/index.js`

File: assets/css/index.css [hunk 4]
```diff
@@ -227,6 +227,7 @@ hr {
   word-wrap: break-word;
   font-family: 'Syne', sans-serif;
 } 
+
```
Comment (shahednasser) [inline assets/css/index.css:230]: This file shouldn't be changed. Return it to how it was with `git checkout assets/css/index.css`

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by shubhampandeyhld** [comment]
I need again forked the repo. because its saying your branch is up to date with master

**Comment by shahednasser** [comment]
closed by #75

**Pull request closed without merging**